updater/lib/dependabot/api_client.rb

```diff
@@ -52,10 +52,10 @@ def fetch_job
       Job.new(job_data.merge(token: job_token))
     end
 
-    def create_pull_request(dependencies, updated_dependency_files,
-                            base_commit_sha, pr_message)
+    # TODO: Make `base_commit_sha` part of Dependabot::DependencyChange
```

Comment:
I've punted on this for now because how we obtain the `base_commit_sha` is a little unusual. I think for the purposes of `Dependabot::Updater` it should always be present on the `job`, but I need to verify this.